host-controller/src/main/java/org/jboss/as/host/controller/HostControllerService.java

```diff
@@ -99,8 +101,10 @@ public HostControllerService(final HostControllerEnvironment environment, final
 
     @Override
     public void start(StartContext context) throws StartException {
-        //Moved to AbstractControllerService.start()
-        // processState.setStarting();
+
+        // If this is a reload, track start time independently of the overall process elapsed time
+        ElapsedTime startupTime = everStopped ? elapsedTime.checkpoint() : elapsedTime;
```

Comment:
It looks like `everStopped` should be flagged as true when HostControllerService is stopped, otherwise this ternary operator makes no sense.

Reply:
Fixed